.github/actions/playwright-atomic/action.yml

```diff
@@ -22,12 +22,19 @@ runs:
       working-directory: packages/atomic
       shell: bash
     - name: Run Playwright tests
+      id: playwright
       run: npx playwright test ${{ inputs.testsToRun }} --shard=${{ inputs.shardIndex }}/${{ inputs.shardTotal }}
       working-directory: packages/atomic
       shell: bash
+      continue-on-error: true
     - uses: actions/upload-artifact@4cec3d8aa04e39d1a68397de0c4cd6fb9dce8ec1 # v4
       if: ${{ inputs.uploadArtifacts == 'true' }}
       with:
         name: blob-report-${{ matrix.shardIndex }}
         path: packages/atomic/blob-report
         retention-days: 5
+    - run: |
+        if [[ ${{ steps.playwright.outcome}} == 'failure' ]];
```

Comment:
`steps.*.outcome`, see https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#steps-context

Reply:
It's the kind of thing that seems obvious today, but will seem less so if we have to modify the action 6 months from now, so I think it'ws worth adding the link as a comment, yes 😁

Reply:
```suggestion
        //  https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/accessing-contextual-information-about-workflow-runs#steps-context
        if [[ ${{ steps.playwright.outcome}} == 'failure' ]];
```